galsim/randwalk.py

```diff
@@ -133,6 +186,9 @@ def _sbp(self):
     def input_half_light_radius(self):
         """
         The input half-light radius is not necessarily the realized hlr.
+
+        If a profile is sent, this will be -1 if it was a transformation
```

Comment:
I think this is incorrect.  Above, you set it to None if the `half_light_radius` doesn't work.

Reply:
fixed